pkg/controller/utils/utils.go

```diff
@@ -234,8 +240,15 @@ func GetPod(podName string, pods []corev1.Pod) *corev1.Pod {
 
 // LabelsForAerospikeCluster returns the labels for selecting the resources
 // belonging to the given AerospikeCluster CR name.
-func LabelsForAerospikeCluster(name string) map[string]string {
-	return map[string]string{"app": "aerospike-cluster", "aerospike-cluster_cr": name, "podName": ""}
+func LabelsForAerospikeCluster(clName string) map[string]string {
+	return map[string]string{"app": "aerospike-cluster", "aerospike-cluster_cr": clName}
```

Comment:
Change label keys in  "aerospike.com/<key>" form 
"aerospike.com/cr"
"aerospike.com/rack-label